<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="ru.stqa.pft.rest.RestTests" tests="1" skipped="0" failures="1" errors="0" timestamp="2023-04-14T15:08:28" hostname="BEST-KOMP" time="0.753">
  <properties/>
  <testcase name="testCreateIssue" classname="ru.stqa.pft.rest.RestTests" time="0.753">
    <failure message="org.apache.http.client.HttpResponseException: status code: 404, reason phrase: Not Found" type="org.apache.http.client.HttpResponseException">org.apache.http.client.HttpResponseException: status code: 404, reason phrase: Not Found
	at app//org.apache.http.impl.client.AbstractResponseHandler.handleResponse(AbstractResponseHandler.java:70)
	at app//org.apache.http.client.fluent.Response.handleResponse(Response.java:91)
	at app//org.apache.http.client.fluent.Response.returnContent(Response.java:98)
	at app//ru.stqa.pft.rest.RestTests.getIssues(RestTests.java:30)
	at app//ru.stqa.pft.rest.RestTests.testCreateIssue(RestTests.java:21)
	at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base@17.0.6/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@17.0.6/java.lang.reflect.Method.invoke(Method.java:568)
	at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
	at app//org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
	at app//org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
	at app//org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
	at app//org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
	at app//org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
	at app//org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
	at app//org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
	at java.base@17.0.6/java.util.ArrayList.forEach(ArrayList.java:1511)
	at app//org.testng.TestRunner.privateRun(TestRunner.java:806)
	at app//org.testng.TestRunner.run(TestRunner.java:601)
	at app//org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
	at app//org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
	at app//org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
	at app//org.testng.SuiteRunner.run(SuiteRunner.java:330)
	at app//org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at app//org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
	at app//org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
	at app//org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
	at app//org.testng.TestNG.runSuites(TestNG.java:1099)
	at app//org.testng.TestNG.run(TestNG.java:1067)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.runTests(TestNGTestClassProcessor.java:146)
	at org.gradle.api.internal.tasks.testing.testng.TestNGTestClassProcessor.stop(TestNGTestClassProcessor.java:91)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:61)
	at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@17.0.6/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base@17.0.6/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@17.0.6/java.lang.reflect.Method.invoke(Method.java:568)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$3.run(TestWorker.java:193)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:133)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:71)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
</failure>
  </testcase>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
